### PR TITLE
Update motor mappings for new wrist

### PR DESCRIPTION
**Pull request by abhaybd**
We no longer have a differential so this removes those values and updates usages.

### CODE REVIEW

File: src/world_interface/data.h
Comment (huttongrabiel) [inline src/world_interface/data.h:66]: **non-blocking nitpick:** Do we want jointid_t and mountedperipheral_t to be single line enums or is it clang-format's doing? If single line enums are fine, it would be nice to add a newline between jointid_t and all_jointid_t for readability. If it's clang-format doing it, there is an option we can set in clang-format to enforce multi line enums.

**Review by huttongrabiel — DISMISSED** [review]
Looks good. Might want to wait for one more review from somebody with more knowledge of the code base than me.

File: src/world_interface/data.h
Comment (abhaybd) [reply src/world_interface/data.h]: Good point, multiline enums are more readable imo. Fixed it with new clang-format rule.

**Review by huttongrabiel — APPROVED** [review]

**Pull request merged**
